build_unit_tests.py

```
@@ -0,0 +1,86 @@
+        2. All code should be in the same namespace.
+        3. The entire code output should be wrapped in a single set of ```csharp tags.
+
+        Remember:
+        - Strictly adhere to the knowledge base at all times.
+        - Show your thought process in the <analysis> tags.
+        - Ensure all code is well-organized and properly structured.
+        - Include any necessary DTOs and Factory classes in your output.
+
+        Please begin your analysis now.
+        ```
+    """.format(knowledge_base_content=knowledge_base_content, unit_testing_engine=unit_testing_engine, function=function, sut=sut, test_cases=test_cases, additional_information=additional_information, relevant_files=formatted_file_contents)
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)